### PR TITLE
[4.0] com_privacy filters

### PR DESCRIPTION
**Pull request by brianteeman**
Make sure that all the filters have a valid label in the requests and consents views

### CODE REVIEW

File: administrator/components/com_privacy/forms/filter_requests.xml [hunk 1]
```diff
@@ -12,8 +12,7 @@
 		<field
 			name="status"
 			type="requeststatus"
-			label="COM_PRIVACY_FILTER_STATUS"
```
Comment (Quy) [inline administrator/components/com_privacy/forms/filter_requests.xml:15]: Remove language strings.

File: administrator/components/com_privacy/forms/filter_requests.xml [hunk 2]
```diff
@@ -22,8 +21,7 @@
 		<field
 			name="request_type"
 			type="requesttype"
-			label="COM_PRIVACY_FILTER_REQUEST_TYPE"
```
Comment (Quy) [inline administrator/components/com_privacy/forms/filter_requests.xml:25]: Remove language strings.

File: administrator/components/com_privacy/forms/filter_requests.xml (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_privacy/forms/filter_requests.xml:15]: Sorry but thats not possible - it doesnt exist ;)

File: administrator/components/com_privacy/forms/filter_requests.xml (hunk 2 above)
Comment (brianteeman) [reply administrator/components/com_privacy/forms/filter_requests.xml:25]: Sorry but thats not possible - it doesnt exist ;)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c0070619e83cb4a3707c63819dc97b5e1318491d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25824">issues.joomla.org/tracker/joomla-cms/25824</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on c0070619e83cb4a3707c63819dc97b5e1318491d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25824">issues.joomla.org/tracker/joomla-cms/25824</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25824">issues.joomla.org/tracker/joomla-cms/25824</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**